src/api/snap.ts

```diff
@@ -107,6 +107,15 @@ const setSnapVersion = async (provider: EIP1193Provider) => {
   }
 };
 
+const runBackup = async (provider: EIP1193Provider) => {
+  try {
+    await callSnap<SnapVersionResponse>(provider, 'tss_runBackup', null);
+  } catch (e) {
```

Comment:
we should handle error case, otherwise users will keep losing their backups

Reply:
we should send mix panel event when `runBackup` failed